gfx/drivers_context/wayland_ctx.c

```diff
@@ -1423,9 +1464,9 @@ static bool gfx_ctx_wl_has_focus(void *data)
 
 static bool gfx_ctx_wl_suppress_screensaver(void *data, bool state)
 {
-	(void)data;
-	gfx_ctx_wayland_data_t *wl = (gfx_ctx_wayland_data_t*)data;
-	
+        (void)data;
```

Comment:
This is not C89-compliant. Declare variables at the top of a function or code block.